docs/share_channels.rst

```diff
@@ -56,6 +50,16 @@ Make Content Channels Available for Import into Kolibri
 
          Channel ID/token number in the My Channels page.
 
+-  **Using this channel** tab in the extended channel information panel
```

Comment:
Could you also add that this tab only appears if the channel has been published?